Add-on Store: Added the action to cancel the install of a add-on when it is being downloaded. (#16671)
nvaccess/nvda: user_docs/en/userGuide.md

You can install and update add-ons by [browsing Available add-ons](#AddonStoreBrowsing).
Select an add-on from the "Available add-ons" or "Updatable add-ons" tab.
Then use the update, install, or replace action to start the installation.

> It is also possible to cancel the install just after the download.

Reviewer: ```suggestion
It is also possible to cancel the install before exiting the Add-on Store.
```
